src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -81,17 +81,28 @@ def _convert_run_to_spans(
         }.items():
             attributes.update(zip(io_attributes, _convert_io(run.get(io_key))))
         attributes.update(_prompt_template(run["serialized"]))
+        events: List[SpanEvent] = []
+        if (error := run["error"]) is None:
+            status_code = SpanStatusCode.OK
+        else:
+            status_code = SpanStatusCode.ERROR
+            error_event = next(filter(lambda event: event["name"] == "error", run["events"]))
+            events.append(
+                SpanException(
+                    message=error,
+                    timestamp=error_event["time"],
+                )
+            )
```

Comment:
doesn't this only record one event but in reality we might want to record all the exceptions? Also please feel free to extend the event with callstack etc. as noted in the spec.

Reply:
you're right i shouldn't assume there's only one exception. i updated the code. on the other hand, we don't have access to the call stack with the current approach which is to simply transcribe the final langsmith result. To get to the actual call stack we would need to more "high touch" version, i.e. making and keeping our spans in a stack and so on.

Reply:
actually, i think the langsmith structure only keep a single copy of the error message, so if there are more than one error event, i don't see that they would have separate messages

Reply:
added a comment to clarify why only one error event is kept